src/controllers/userController.js

```diff
@@ -95,5 +95,52 @@ class register {
       });
     }
   }
+
+//updating driver or operator profile
+
+static async updateProfile (req, res){
+  try {
+    
+    const {email}=req.user;
+    const [ updatedField ] = await Users.update(req.body, {
+      
+      where: { email }
+    });
+    
+    if (updatedField) {
+      const updatedUser = await Users.findOne({ where: { email },
+        attributes:{
+          exclude:['password']
+        }
+      });
+
+      return res.status(200).json({ 
+        status:'successfull updated',
+        user: updatedUser })
```

Comment:
you first need to check if the user exists 
then if he does then update the user 
or return 404